data/remnant/remnant 2 side missions.txt

```diff
@@ -144,3 +144,60 @@ mission "Remnant: Visit to Paradise 1"
 			label mainreturn
 			`	Taely leads you around the area for a few more hours, pointing out historical features, places where they resurrected trades and skills from before the advent of flight, and other interesting things. Eventually she heads back to the train station, which deposits you back in the main spaceport.`
 			`	"I hope you enjoyed the tour, <first>. It was interesting to see our history through fresh eyes." She smiles mischievously. "And I enjoyed the hoverboarding, too. Maybe we can do it again sometime." She pauses for a moment, focusing on something you cannot see. "Looks like duty calls. Embers guide your way, Captain <last>." She waves, and heads off into the crowd.`
+
+
+
+mission "Remnant: Teciimach Deployment"
+	name "New EMP deployment"
+	description "The Remnant are deploying a new version of the EMP, and invite you to watch."
+	invisible
+	minor
+	landing
+	source
+		attributes "shipyard" "remnant"
+	to offer
+		has "event: remnant: cognizance calm"
+		has "event: remnant research update bjd1"
+	on offer
+		event "remnant: teciimach deployment"
+		conversation
+			`You note a disturbance in the shipyards before the <ship> even touches the ground. Every pad is filled with ships, camels pulling wagon trains loaded with canisters trundle out of the outfitter, and technicians clamber everywhere. As you step out of your ship, Taely slides down the side of a Merganser resting next to you.`
+			`	"Good timing, Captain! We just rolled out an upgrade to some of our oldest equipment, and our Mergansers finally have something to carry as first-strike weapons. Interested in taking a look?"`
+			choice
+				`	"Yes."`
+				`	"Sorry, but I'm a bit busy at the moment."`
+					goto empno
+			`	She turns and heads towards a nearby lift where another crew of Remnant had just moved several EMP torpedo canisters into place. "Well, you have probably seen us use EMP torpedoes now and then, right? Well, as you probably noticed, they have a few tactical issues. They are slow, they often hit our ships almost as much as our targets, and they take up space we could otherwise be using for more important things like laboratories or scanners." She taps a few commands into the console, and the lift begins raising the canisters towards the waiting ship.`
+			`	"Over the past decades we have had a few teams working on improving them from different angles. Some focused on the torpedoes themselves, others focused on the launcher. The surprising development, however, was from a team working down in the Warren. They figured out a way to shave a few cubic meters off the systems that assemble the torpedoes, and compress the whole structure. Very impressive work, and the results should be in the outfitter for you in the next day or two."`
```

Comment:
One last slight change in how Taely describes the changes made to the weapon to account for the suggested stats change to lower the capacity to 6.
```suggestion
			`	"Over the past decades we have had a few teams working on improving them from different angles. Some focused on the torpedoes themselves, others focused on the launcher. The surprising development, however, was from a team working down in the Warren. They figured out a way to shave a few cubic meters off the systems that assemble the torpedoes, and compress the whole structure. Unfortunately it carries a few less torpedoes, but that is more than made up for by the improvements to the munitions, which are better than their predecessor in almost every way. Very impressive work, and the results should be in the outfitter for you in the next day or two."`
```